### DISCUSSION TITLE
How to do step migration in adonis 5 like in laravel?

### DISCUSSION THREAD

**Discussion by tsulatsitamim** [adonisjs/core#4177] [Help]
How to do step migration in adonis 5 like in laravel?

**Comment by MaximeMRF** [comment]
Adonis doesn't support the step flag when you migrate your db with ace. There is only the [batch](https://docs.adonisjs.com/guides/database/migrations#rollback-migrations) flag like laravel but the behavior is different of the laravel step flag.

**Comment by MaximeMRF** [comment]
Hello @tsulatsitamim,

I have open a [pull request](https://github.com/adonisjs/lucid/pull/940) to add the step option to the migrator, feel free to describes your use cases in a comment on this pr.
